### DISCUSSION TITLE
Custom 404 page not picked up?

### DISCUSSION THREAD

**Discussion by wereHamster** [vercel/next.js#11544] [Help]
https://codesandbox.io/s/nextjs-933-custom-404-page-x6r6f

When you go to /404, then 404.js page is rendered. When you go to /doesnotexist, then the standard Next.js 404 page is rendered.

I'm not sure what else I'm supposed to do. The documentation (https://nextjs.org/docs/advanced-features/custom-error-page#404-page) says to create `pages/404.js` which is what I did.

**Comment by timneutkens** [comment]
<img width="1370" alt="Screen Shot 2020-04-01 at 11 40 47" src="https://user-images.githubusercontent.com/6324199/78122964-c3682280-740d-11ea-9fc0-931aebaaa5dc.png">

Works perfectly in the codesandbox example that you're giving 🤔

**Comment by wereHamster** [comment]
Weird. If you start with the next.js hello world template on CSB, add a 404 page, you can reproduce the behaviour I've observed. Perhaps it works in my example because I restarted the CSB container…
